sycl/source/detail/scheduler/graph_builder.cpp

```diff
@@ -264,7 +264,7 @@ Command *Scheduler::GraphBuilder::insertMemoryMove(MemObjRecord *Record,
     AllocaCmdSrc = (Record->MAllocaCommands.end() != It) ? *It : nullptr;
   }
   if (!AllocaCmdSrc)
-    throw runtime_error("Cannot find buffer allocation");
+    throw runtime_error("Cannot find buffer allocation", PI_INVALID_VALUE);
```

Comment:
I don't think `PI_INVALID_VALUE` is appropriate here, but this and some other exceptions here are internal errors and should probably be assertions instead, but this is outside the scope of this patch, so I'm fine with this resolution for now.